### ISSUE TITLE
Question: Is there a way to know which drop zone an item came from?

### ISSUE THREAD

**Issue by PaulBeaudet** [isaacHagoel/svelte-dnd-action#258]
Feels like I'm probably missing something obvious here.

Say I had a to-do and completed column each with the same type drop zone. With those columns would be cards that could be dragged and dropped within the columns or between them.

I'm trying to figure the origin of the dragged item so I can trigger various events depending on the column dragged into.

Is there anything in the finalize or consider events that show something identifiable from the origin (dragstart) drop zone?

**Comment by isaacHagoel** [comment]
it should be easy enough. The idea is to use the trigger provided in the
event and store the information of the zone the event fired on.
this can be done in various ways depending on how your logic is split into
components.
feel free to make a REPL that proximates your usecase and i can try to show
you what i mean.

**Comment by PaulBeaudet** [comment]
I ended up figuring this out by creating a ghost of transitional id arrays for each column that store the state from the initial pick-up. Given I was relying on in-memory state to figure this out, I think the problem that I was running into had to do with wrong assumptions about svelte writable stores. I didn't realize that states were two-way bound even as the export variable of a component. It does make sense in retrospect.

so if one wrote something like this with a writable store 
`<myComponet list={$writeableArrayOfCards} />`

and then inside of myComponent was written
`
 const onConsider = (e) => {
    list = e.detail.items;
  };
  const onFinalize = (e) => {
    // external logic using .update on store
    rearrangeList(e.detail.items, e.detail.info.id); 
  };
`
If one was expecting "list" was one-way bound / internally scoped to the component they would be in for confusion.
On consider would constantly change "list" to reflect the lastest location, not necessarily the original target in the case of multiple lists. As such, it's probably going to show event data from the column/list that is about to be dropped into.

I did look through the event passed from on:consider and on:finalize quite a bit. I found a source id prop, but I might have been too focused on making things happen in on:consider to realize how transient it was or wasn't in on finalize. 

Maybe I'll get my code on GitHub eventually. Others will likely have better ideas than what I did.
